src/main/resources/db/migration/V6_0__create_table_sykmelding.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE sykmelding (
+    id                    SERIAL         PRIMARY KEY,
+    created_at            TIMESTAMPTZ    NOT NULL,
+    referanse_uuid        CHAR(36)       NOT NULL,
+    personident           VARCHAR(11)    NOT NULL,
```

Comment:
Noen spørsmål og tanker her:
1. Burde vi hatt en foreign key til personoppgave-id? Man kunne hatt det i stedet for referanse-uuiden til sykmeldingen, selv om det gir jo kanskje mening å lagre den fordi denne faktisk heter sykmelding 🤔
2. Trenger vi egentlig å lagre ned `personident`? Hvis man alltid har en `sykmelding` som er knyttet til en `personoppgave` så kunne man joinet for å finne den informasjonen derfra også. Blir vel en avveining om dataminimering vs enklere query.
3. Burde man hatt en `uuid` her som vi pleier i de andre tabellene våre, eller blir det litt overflødig når vi har `id`? Marius diskuterte noe lignende før jul om hva som er formålet med `uuid` i våre tabeller.